packages/thumbprint-react/components/ModalDefault/index.jsx

```diff
@@ -160,6 +160,21 @@ const modalContentPropTypes = {
     children: PropTypes.node.isRequired,
 };
 
+const modalContentFullBleedPropTypes = {
+    /**
+     * Content (usually a form) that makes up the main part of the modal.
+     */
+    children: PropTypes.node.isRequired,
+    /**
+     * Allows the React `className` prop to be passed through to the rendered element.
+     */
+    className: PropTypes.string.isRequired,
```

Comment:
`className` and `style` shouldn't be required because you can use one but not the other.

Reply:
done